lib/private/Files/External/FrontendDefinitionTrait.php

```diff
@@ -84,7 +84,7 @@ public function addParameter(DefinitionParameter $parameter) {
 	}
 
 	/**
-	 * @return string[]
+	 * @return string[]|array
```

Comment:
I wonder what is the "correct" way to specify this?
Maybe:
```suggestion
	 * @return string[]|string[][]
```
IMO it can be an "array of strings" or "an array of arrays of strings"

But maybe what is here is enough.

Reply:
both are same! IMO array is more readable. Whay do you think?